features/film/model/__tests__/overlay-label.test.ts

```
@@ -0,0 +1,62 @@
+  return {
+    setNumber,
+    totalSets,
+    percentage,
+    reps: { type: "simple", reps },
+    prescribedKg: null,
+  };
+}
+
+describe("formatOverlayLabel", () => {
+  it("동작·무게·퍼센트·세트를 '종목 · kg · p% · Set n/m' 형식으로 반환한다", () => {
+    const result = formatOverlayLabel(movement("Back Squat"), setPlan(3, 5, 2, 85), 100);
```

ESLint

[error] Replace `movement("Back·Squat"),·setPlan(3,·5,·2,·85),·100` with `⏎······movement("Back·Squat"),⏎······setPlan(3,·5,·2,·85),⏎······100⏎····`

(prettier/prettier)